tests/libtest/test_provider_create_hosted_cluster.py

```diff
@@ -122,6 +132,76 @@ def test_storage_client_connected(self):
         cluster_names = list(config.ENV_DATA["clusters"].keys())
         assert HostedODF(cluster_names[-1]).get_storage_client_status() == "Connected"
 
+    @runs_on_provider
+    @hci_provider_required
+    def test_create_hosted_cluster_with_fixture(
+        self, create_hypershift_clusters, destroy_hosted_cluster
+    ):
+        """
+        Test create hosted cluster with fixture
+        """
+        log_step("Create hosted client")
+        cluster_name = get_random_hosted_cluster_name()
+        odf_version = str(get_ocs_version_from_csv()).replace(".stable", "")
+        if "rhodf" in odf_version:
+            odf_version = get_odf_tag_from_redhat_catsrc()
+        ocp_version = get_latest_release_version()
+        nodepool_replicas = 2
+
+        create_hypershift_clusters(
+            cluster_names=[cluster_name],
+            ocp_version=ocp_version,
+            odf_version=odf_version,
+            setup_storage_client=True,
+            nodepool_replicas=nodepool_replicas,
+        )
+
+        log_step("Switch to the hosted cluster")
+        ocsci_config.switch_to_cluster_by_name(cluster_name)
+
+        server = str(OCP().exec_oc_cmd("whoami --show-server", out_yaml_format=False))
+
+        assert (
+            cluster_name in server
+        ), f"Failed to switch to cluster '{cluster_name}' and fetch data"
+
+    @runs_on_provider
+    @hci_provider_required
+    def test_create_destroy_hosted_cluster_with_fixture(
+        self, create_hypershift_clusters, destroy_hosted_cluster
+    ):
+        """
+        Test create hosted cluster with fixture
```

Comment:
Docstring is identical to the one of the previous test. I'd write something like "create and destroy hosted cluster", to avoid confusion.

Reply:
done